stdlib/SparseArrays/test/sparse.jl

```diff
@@ -2253,4 +2253,22 @@ end
     @test SparseMatrixCSC(transpose(A)) isa SparseMatrixCSC
 end
 
+@testset "Issue #28369" begin
+    M = reshape([[1 2; 3 4], [9 10; 11 12], [5 6; 7 8], [13 14; 15 16]], (2,2))
+    MP = reshape([[1 2; 3 4], [5 6; 7 8], [9 10; 11 12], [13 14; 15 16]], (2,2))
+    S = sparse(M)
+    SP = sparse(MP)
+    @test isa(transpose(S), Transpose)
+    @test transpose(S) == copy(transpose(S))
+    @test Array(transpose(S)) == copy(transpose(M))
+    @test permutedims(S) == SP
+    @test permutedims(S, (2,1)) == SP
+    @test permutedims(M, (1,2)) == S
```

Comment:
Did you perhaps mean to test `permutedims(S, (1,2)) == M` instead?

Reply:
`permutedims(M, (1,2)) == S`  doesn't test the new code. `permutedims(S, (1,2)) == M` is definitely better.

Reply:
I fixed this and rebased and pushed the PR again.